app/assets/javascripts/discourse/widgets/post-cooked.js.es6

```diff
@@ -17,6 +17,10 @@ export default class PostCooked {
     this.expanding = false;
     this._highlighted = false;
     this.decoratorHelper = decoratorHelper;
+    this.currentUser = decoratorHelper.widget.currentUser;
+    if (this.currentUser) {
```

Comment:
I don't think this test is needed since you're only setting ignoredUsers when there is a currentUser?